CHANGELOG.md

```diff
@@ -4,6 +4,22 @@
 All notable changes to this project will be documented in this file.
 The CHANGELOG is available on [Github](https://github.com/luc-tielen/souffle-haskell.git/CHANGELOG.md).
 
+## [0.2.0] - 2020-04-22
```

Comment:
What are you implying here? That this be released in 3 days.. or else? 😝

Reply:
It just pure hope and 22nd is some anniversary for me, hoping for a gift ;)

Reply:
Haha, should be possible I think. 😄